runtime/moonbase/tests/integration_test.rs

```diff
@@ -2444,6 +2444,40 @@ fn refund_ed_0_evm() {
 		});
 }
 
+#[test]
+fn author_does_not_receive_priority_fee() {
+	ExtBuilder::default()
+		.with_balances(vec![(
+			AccountId::from(BOB),
+			(1 * UNIT) + (21_000 * (500 * GIGAWEI)),
+		)])
+		.build()
+		.execute_with(|| {
+			// Some block author as seen by pallet-evm.
+			let author = AccountId::from(<pallet_evm::Pallet<Runtime>>::find_author());
+			// Currently the default impl of the evm uses `deposit_into_existing`.
+			// If we were to use this implementation, and for an author to receive eventual tips,
+			// the account needs to be somehow initialized, otherwise the deposit would fail.
+			Balances::make_free_balance_be(&author, 100 * UNIT);
+
+			// EVM transfer.
+			assert_ok!(Call::EVM(pallet_evm::Call::<Runtime>::call {
+				source: H160::from(BOB),
+				target: H160::from(ALICE),
+				input: Vec::new(),
+				value: (1 * UNIT).into(),
+				gas_limit: 21_000u64,
+				max_fee_per_gas: U256::from(300 * GIGAWEI),
+				max_priority_fee_per_gas: Some(U256::from(200 * GIGAWEI)),
+				nonce: Some(U256::from(0)),
+				access_list: Vec::new(),
+			})
+			.dispatch(<Runtime as frame_system::Config>::Origin::root()));
+			// Author free balance didn't change.
+			assert_eq!(Balances::free_balance(author), 100 * UNIT,);
```

Comment:
You could probably also assert that the total supply has shrunk by the priority fee amount

Reply:
Added a test to verify that the total supply shrunk by 80% of the total (base + priority) fee.